packages/canvas/package.py

```diff
@@ -76,3 +76,7 @@ def setup_build_environment(self, env):
         env.prepend_path("PATH", os.path.join(prefix, "bin"))
         # Cleanup.
         sanitize_environments(env, "PATH")
+
+    def setup_run_environment(self, run_env):
+        run_env.prepend_path("ROOT_INCLUDE_PATH", self.prefix.include)
+        run_env.append_path("CET_PLUGIN_PATH", self.prefix.lib)
```

Comment:
`canvas` should not need to affect these variables.  I'd remove `setup_run_environment` here.